app/views/games/show.html.haml

```diff
@@ -109,8 +108,8 @@
       - prevscore = -1
       - currentnum = 1
       - countnum = 1
-      -all_players = @game.registrations.group_by(&:display_faction_id).sort_by{|x| x.display_score}.reverse_each do |p|
-    %div.humans.clear{:style=>"float:left;width:40%"}
+      -all_players = @game.registrations.group_by(&:display_faction_id).sort_by{|x| x.display_score}
+    %div.humans.clear{:style=>"float:left;width:350px"}
```

Comment:
Are we positive this works well regardless of device and screen size
Also, inline styles are bad, can we throw this in `app/assets/stylesheets/scoreboard.css.scss`